processing/src/main/java/org/apache/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -110,6 +110,13 @@
   public static final byte HLL_SKETCH_TO_STRING_CACHE_TYPE_ID = 0x31;
   public static final byte HLL_SKETCH_TO_ESTIMATE_AND_BOUNDS_CACHE_TYPE_ID = 0x32;
 
+  // Quantiles sketch in momentsketch extension
+  public static final byte MOMENTS_SKETCH_BUILD_CACHE_TYPE_ID = 0x33;
+  public static final byte MOMENTS_SKETCH_MERGE_CACHE_TYPE_ID = 0x34;
+  public static final byte MOMENTS_SKETCH_TO_QUANTILES_CACHE_TYPE_ID = 0x35;
```

Comment:
The quantiles/min/max are post-aggs, so those IDs should be defined in `PostAggregatorIds` instead